ignite/services/network/networktypes/profile.go

```diff
@@ -18,12 +18,12 @@ type Validator struct {
 }
 
 func (v Validator) ToProfile(
-	campaignID uint64,
+	projectID uint64,
 	vouchers sdk.Coins,
 	shares campaigntypes.Shares,
 ) Profile {
 	return Profile{
-		CampaignID:      campaignID,
+		CampaignID:      projectID,
```

Comment:
Btw, I'm not even sure the reason for having `CampaignID` in a `Profile` object

We should check if this is still something necessary and up to date